system/HTTP/URI.php

```diff
@@ -690,6 +708,8 @@ public function setAuthority(string $str)
      * @see https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml
      *
      * @return $this
+     *
+     * @TODO PSR-7: Should be `withScheme($scheme)`.
```

Comment:
Why not go ahead and add the new methods as aliases and deprecate these?

Reply:
Added.